pkg/features/kube_features.go

```diff
@@ -269,6 +269,13 @@ const (
 	// Enable ServiceAccountTokenVolumeProjection support in ProjectedVolumes.
 	TokenRequestProjection utilfeature.Feature = "TokenRequestProjection"
 
+	// owner: @mikedanese
+	// alpha: v1.13
+	//
+	// Migrate ServiceAccount volumes to use a projected volume consisting of a
+	// ServiceAccountTokenVolumeProjection.
```

Comment:
worth noting here that enabling this feature will add new required flags in the apiserver

Reply:
I mentioned that new flags are required, but not which ones. Want me to add that too?